Demo/Demo/Extensions/PaymentButtonEnums+Extension.swift

```diff
@@ -94,3 +94,19 @@ extension PaymentButtonFundingSource {
         PaymentButtonFundingSource.allCases().map { $0.rawValue }
     }
 }
+
+extension PayPalButton.Label {
+    
+    static func allCases() -> [PayPalButton.Label] {
+        [
+            PayPalButton.Label.none,
+            PayPalButton.Label.checkout,
+            PayPalButton.Label.buyNow,
+            PayPalButton.Label.payWith
+        ]
+    }
+    
+    static func allCasesAsString() -> [String] {
+        allCases().map { $0.rawValue }
+    }
+}
```

Comment:
If we wanted, we could still use `CaseIterable` here. Looks slightly different in an extension but still pretty clean imo:
```
extension PayPalButton.Label: CaseIterable {

    public static var allCases: [PayPalButton.Label] {
        [.none, .checkout, .buyNow, .payWith]
    }

    static func allCasesAsString() -> [String] {
        Self.allCases.map { $0.rawValue }
    }
}
```

Reply:
@jaxdesmarais oh i tried that but i got a compilation error because the extensions where not on the same file, so i couldnt simply use `allCases`. I'll try this